docs/extending/admin_views.md

```diff
@@ -102,8 +97,21 @@ Now create a `templates/wagtailcalendar/` folder within the `wagtailcalendar` ap
 {% endblock %}
 ```
 
+```css
+/* calender.css */
+table.month {
+            margin: 20px;
+        }
+        table.month td, table.month th {
+            padding: 5px;
+        }
+
+```
+
 Here we are overriding three of the blocks defined in the base template: `titletag` (which sets the content of the HTML `<title>` tag), `extra_css` (which allows us to provide additional CSS styles specific to this page), and `content` (for the main content area of the page). We're also including the standard header bar component, and setting a title and icon. For a list of the recognized icon identifiers, see the [style guide](styleguide).
 
+Note: Moving inline styles to an external file helps improve security by aligning with Content Security Policy (CSP) standards.
```

Comment:
We don't need this, for those that know how CSP works they will know this. 

For those that don't care or know about CSP they will just blindly copy the example and use the more secure approach. 

Where possible we want our docs to be simple and not add cognitive overhead or give people too many options/decisions to make.

Reply:
I understand, thanks